test/compute/2019-12-01/armcompute/zz_generated_containerservices_client.go

```diff
@@ -30,6 +30,9 @@ type ContainerServicesClient struct {
 }
 
 // NewContainerServicesClient creates a new instance of ContainerServicesClient with the specified values.
+// subscriptionID - Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.
```

Comment:
Unfortunate overloading of "credentials". Needs a swagger edit to change?

Reply:
Unfortunately yes.